README.md

```diff
@@ -81,13 +81,24 @@ The sections as enumerated below will walk you through the process of creating a
     node ./deploy.js -b <my-bucket> -s <my-stack> -i <my-docker-image-uri> -r <region>
     ```
    
-   Here is an example:
+    Here is an example:
```

Comment:
Nit: Remove extra space

Reply:
Adding the extra space conforms normal indentation. It also rectifies indentation of image and note below it.